event/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 //! # Ceramic Event
 //! Implementation of ceramic event protocol, with appropriate compatibilility with js-ceramic
 #![warn(missing_docs)]
+
+/// Anchor-related types
+pub mod anchor;
```

Comment:
Why do anchor types live in the `ceramic-event` crate? I would have expected them to live in the `ceramic-anchor-service` crate.